bigquery/schema.go

```diff
@@ -353,16 +353,17 @@ var typeOfByteSlice = reflect.TypeOf([]byte{})
 // (This is the same mapping as that used for RowIterator.Next.) Fields inferred
 // from these types are marked required (non-nullable).
 //
-//	STRING      string
-//	BOOL        bool
-//	INTEGER     int, int8, int16, int32, int64, uint8, uint16, uint32
-//	FLOAT       float32, float64
-//	BYTES       []byte
-//	TIMESTAMP   time.Time
-//	DATE        civil.Date
-//	TIME        civil.Time
-//	DATETIME    civil.DateTime
-//	NUMERIC     *big.Rat
+//		STRING      string
+//		BOOL        bool
+//		INTEGER     int, int8, int16, int32, int64, uint8, uint16, uint32
+//		FLOAT       float32, float64
+//		BYTES       []byte
+//		TIMESTAMP   time.Time
+//		DATE        civil.Date
+//		TIME        civil.Time
+//		DATETIME    civil.DateTime
+//		NUMERIC     *big.Rat
+//	    JSON        map[string]interface{}
```

Comment:
minor nit: looks like comment alignment is odd here?

Reply:
Fixed!